src/test/java/com/epam/tam/unittests/CalculatorTest.java

```diff
@@ -0,0 +1,99 @@
+package com.epam.tam.unittests;
+
+import com.epam.tat.module4.Calculator;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class CalculatorTest {
+
+	private Calculator calculator;
+	private static final double DELTA = 0.01;
+	
+	@BeforeClass
+	public void setUp(){
+		calculator = new Calculator();
+	}
+	
+	@Test(expectedExceptions = IllegalArgumentException.class, dependsOnGroups = {"basic"})
```

Comment:
yes, i listened to the unit test lecuture where you instruct to use @rule, however hometask is done for testNG, there is no @rule, so i did the usual way.